index.html

```diff
@@ -114,13 +114,17 @@ <h4>WorkShop</h4>
         <a href="#">SEE THE WHOLE PROGRAM</a>
       </div>
     </section>
+    
     <section class="speaker-section">
       <div class="text-container">
         <h3>Featured Speakers</h3>
         <hr />
       </div>
       <div class="speaker-grid" id="speaker-container">
-        <!-- Speaker list Will appear here -->
+        <!-- Speaker list Will appear here on mobile -->
+      </div>
+      <div class="speaker-grid-desk" id="desk-speaker">
+        <!-- Speaker list will appear here on desktop  -->
       </div>
 
       <div class="more-btn-container">
```

Comment:
- [ ] While you did a wonderful job creating all your articles dynamically :+1: , for semantics you should use an `article` instead of `div` tag for each element inside the "main program" and "speakers" section

_I place this comment here because there are no changes in the specific section, but you have to look inside this document_